src/archivematicaCommon/lib/elasticsearch/aip_mets_mapping.json

```diff
@@ -0,0 +1,739 @@
+{
+  "properties": {
+    "ns0:mets_dict_list": {
+      "properties": {
+        "ns0:metsHdr_dict_list": {
+          "properties": {
+            "@CREATEDATE": {
+              "type": "date",
+              "format": "dateOptionalTime"
+            }
+          }
+        },
+        "ns0:amdSec_dict_list": {
+          "properties": {
+            "ns0:techMD_dict_list": {
+              "properties": {
+                "ns0:mdWrap_dict_list": {
+                  "properties": {
+                    "@MDTYPE": {
+                      "type": "string"
+                    },
+                    "ns0:xmlData_dict_list": {
+                      "properties": {
+                        "ns4:object_dict_list": {
+                          "properties": {
+                            "ns4:originalName": {
+                              "type": "string"
+                            },
+                            "@xsi:type": {
+                              "type": "string"
+                            },
+                            "ns4:objectCharacteristics_dict_list": {
+                              "properties": {
+                                "ns4:size": {
+                                  "type": "string"
```

Comment:
This was autodetected by ES as a string, which suggests that non-obviously numberical fields can end up here.